packages/prismic/src/composables/usePrismic/index.ts

```diff
@@ -14,23 +14,18 @@ interface UsePrismic {
   search: Search;
 }
 
-export default function usePrismic(cacheId?: string): UsePrismic {
-  const { initialState, saveToInitialState } = useSSR(cacheId);
-  const loading = ref(false);
-  const error = ref(null);
-  const doc: Ref<Document | Document[]> = ref(initialState || {});
-  const meta: Ref<PrismicMeta | null> = ref(null);
+export default function usePrismic(): UsePrismic {
```

Comment:
add an id here, otherwise the data will be shared between each `usePrismic` integration

Reply:
Now it is:
```ts
(cacheId: string): UsePrismic
```